api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -47,7 +47,12 @@ public String readMandrillApiKey() {
 
   @Override
   public String getMoodleApiKey() {
-    return getCredentialsBucketString("moodle-key.txt");
+    if(configProvider.get().featureFlags.enableMoodleV2Api) {
+      return getCredentialsBucketString("moodle.txt");
```

Comment:
Please put these constants into a config file, and also rename them to indicate v1 vs v2 somehow.

Reply:
We don't yet have application config patterns, which is where these might need to go, or a parent JSON file that individual environments could override. I would still rename the credentials files `moodle-key-v1.txt` and `moodle-key-v2.txt` (or leave the current one alone).

Reply:
Because we push these creds manually to google buckets I have to leave the current one alone for the meantime. I can rename the new one v2 though.